src/Deserialization/Transaction.js

```diff
@@ -167,6 +201,7 @@ exports._unpackCostModel = cm => {
         catch(_){
             err=true;
         }
+        op++;
```

Comment:
Little bit more idiomatic

```suggestion
    try {
      for(var op = 0;; op++) {
        res.push(cm.get(op).to_str());
      }
    } catch { }
```